tox.ini

```diff
@@ -101,6 +101,15 @@ deps =
     .[opentelemetry]
 commands = {env:SNOWFLAKE_PYTEST_DAILY_CMD} -vvv -m "integ or unit" {posargs:}  tests
 
+[testenv:modin_extra_without_pandas_extra]
+allowlist_externals = bash
+description = SNOW-1345421: test that we can use modin by specifying the modin extra but not the pandas extra.
+deps =
+    .[development]
```

Comment:
@sfc-gh-nkrishna we should make it more compact in your PR right?

Reply:
Yes

Reply:
yeah after that PR the only `deps` here should be `[modin-development]`